File: python language/Bill.py
class Bill:
    def __init__(self, title, bill_id, status):
        self.title = title
        self.status = status
        self.bill_id = bill_id
        self.votes = 0

    def get_title(self):
        return self.title

    def set_title(self, title):
        self.title = title

    def get_status(self):
        return self.status

    def set_status(self, status):
        self.status = status

    def get_votes(self):
        return self.votes

    def set_votes(self, votes):
        self.votes = votes

    def get_bill_id(self):
        return self.bill_id

    def set_bill_id(self, bill_id):
        self.bill_id = bill_id

    def vote(self):
        print(f"Voting on bill '{self.title}'")
        self.votes += 1

File: python language/Bill.java
public class Bill implements Votable {
    private String title;
    private String status;
    private int votes;
    private int billId;

    public Bill(String title, int billId,String status) {
        this.title = title;
        this.status = status;
        this.billId=billId;
        this.votes = 0;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public int getVotes() {
        return votes;
    }

    public void setVotes(int votes) {
        this.votes = votes;
    }
    public int getBillID() {
        return billId;
    }

    public void setBillID(int billId) {
        this.billId = billId;
    }

    @Override
    public void vote() {
        System.out.println("Voting on bill '" + title + "'");
        votes++;
    }
}
